docs/pipeline-components-and-applications/enrichment-components/enrich-nsq/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Enrich NSQ (cloud agnostic)"
+date: "2022-10-03"
```

Comment:
```suggestion
date: "2023-04-25"
```

Reply:
This field does not do anything in Docusaurus (Docusaurus gets the date from git), we can just remove it. I didn’t do that because I didn’t want to bump the last edit date in git on all files.